x-pack/plugins/security_solution/public/resolver/view/panels/use_format_date.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment-timezone';
+import { useUiSetting } from '../../../../../../../src/plugins/kibana_react/public';
+import { formatter, invalidDateText } from './panel_content_utilities';
+
+export function useDateFormatSetting(
+  timestamp: ConstructorParameters<typeof Date>[0] | undefined
+): string | null {
+  const dateFormatSetting: string = useUiSetting('dateFormat');
+  const timezoneSetting: string = useUiSetting('dateFormat:tz');
+
+  if (timestamp === undefined) return null;
+
+  const date = new Date(timestamp);
+  if (date && isFinite(date.getTime())) {
```

Comment:
:grey_question: Should this be `Number.isFinite` instead of the global one?

Reply:
Yea, thanks

Reply:
That wasn't a passive-aggressive "I think that should be Number.isFinite" btw, I seriously don't know. It might depend on if you want to allow `null`s or something.

Reply:
Oh lol, I didn't take it as passive aggressive 😂, it just should be since we cast it to a date beforehand so `getTime()` should return a number if it's a valid date. Just realized I didn't update this either. Will do and get a PR up tomorrow